### ISSUE TITLE
suport for mariadb 10.1

### ISSUE THREAD

**Issue by tolomaus** [zendesk/maxwell#456]
Hi,

I just set up a simple mariadb 10.1 => maxwell => kafka integration and it appears to work just fine: I can see table update logs appear in the kafka topic.

I don't see mariadb 10.1 being officially supported though by maxwell, so I wonder if there are any issues I may expect going forward?

Thanks in advance

**Comment by osheroff** [comment]
Dunno.  Will leave this issue open to any watches who might chime in to see if anyone's run maxwell against MariaDB in production.  AFAIK maria's wire-compatibility *should* mean maxwell works just fine (especially if the initial schema-capture works), but you know, the devil's in the details with these kind of things.  

"officially", should work but I haven't heard production reports either way yet.

**Comment by tolomaus** [comment]
Thanks Ben, any issues I discover myself down the wire I will report here.

**Comment by tanji** [comment]
Hi,

MariaDB and MySQL binlog formats are interchangeable. There are some differences down the line but the protocol is the same, so as long as you don't parse things such as GTID's etc. you should be fine.

**Comment by duraki** [comment]
@tanji that is correct but officially, MariaDB is not supported. For everyone else, check issue #769
